projects/cdk/observables/drag-and-drop-from.ts

```diff
@@ -16,7 +16,7 @@ export class TuiDragState {
     constructor(readonly stage: TuiDragStage, readonly event: MouseEvent) {}
 }
 
-export function dragAndDropFrom(element: Element): Observable<TuiDragState> {
+export function dragAndDropFrom(element: Element): Observable<TuiDragState> | never {
```

Comment:
Why is that `never`? Isn't throwing ensures we return what we expect or we cause an explicit error stating this is not the intended situation?

Reply:
`export function dragAndDropFrom(element: Element): Observable<TuiDragState> | never {` 

in this case, then it would be necessary to rewrite code in such a way that the error will be returned through the stream

<img width="993" alt="image" src="https://user-images.githubusercontent.com/12021443/164180632-58debc62-e2a2-431d-99bc-5ae67d284c33.png">

Reply:
for 

<img width="491" alt="image" src="https://user-images.githubusercontent.com/12021443/164181024-f758e5c1-eaf8-4158-a197-68ad80cecd23.png">

instead of

<img width="523" alt="image" src="https://user-images.githubusercontent.com/12021443/164180843-6cc728c6-ad24-4d95-8bf1-ca4da34b9ab8.png">

Reply:
For external user we are not supposed to know the function throws. It basically serves as a typecast but with an explicit message for the unexpected situation.

Reply:
fixed